components/esp_phy/src/phy_override.c

```diff
@@ -9,8 +9,7 @@
 #include "esp_private/regi2c_ctrl.h"
 #include "esp_private/sar_periph_ctrl.h"
 #include "esp_private/sar_periph_ctrl.h"
-#include "freertos/FreeRTOS.h"
-
+#include <zephyr/kernel.h>
```

Comment:
just a note: we might not be using this file, as per CMakeLists file check

Reply:
I added in the build system. Will update it as well.